src/jquery.maskMoney.js

```diff
@@ -192,10 +193,30 @@
 
                 function mask() {
                     var value = $input.val();
-                    if (settings.precision > 0 && value.indexOf(settings.decimal) < 0) {
+					
+					/**
+					* HTML Support
+					* Added by Tiago Morais
+					* Check if .html() have some number in case of value returns undefined or ""
+					**/
+					if(value==undefined || value==""){
+					var try2=Number($(this).html());				
+						if (typeof try2 === "number") {
+							 $input.val(String(try2));
+							 value=try2;
+						}
+					}
+					
+					if (settings.precision > 0 && String(value).indexOf(settings.decimal) < 0) { 
                         value += settings.decimal + new Array(settings.precision+1).join(0);
                     }
                     $input.val(maskValue(value));
+					$input.html(maskValue(value));
```

Comment:
this line basically breaks functionality for everyone else that is using `input`, so I can't accept it as it is.